tests/test_equilibrium.py

```diff
@@ -181,6 +181,32 @@ def test_eq_change_grid_resolution():
     assert eq.N_grid == 10
 
 
+@pytest.mark.unit
+def test_eq_change_symmetry():
```

Comment:
for this it might be better to test if non-symmetric modes are or aren't in the basis, to make sure we didn't just change the label without modifying stuff.